Doc/library/asyncio-task.rst

```diff
@@ -842,6 +842,12 @@ Task Object
       The *file* argument is an I/O stream to which the output
       is written; by default output is written to :data:`sys.stderr`.
 
+   .. method:: get_coro()
+
+      Return the coroutine object that was passed to the constructor.
```

Comment:
I dislike mentioning `constructor` here.

Above we have the text:
```
   Use the high-level :func:`asyncio.create_task` function to create
   Tasks, or the low-level :meth:`loop.create_task` or
   :func:`ensure_future` functions.  Manual instantiation of Tasks
   is discouraged.
```
Maybe rephrase this line to `Return the coroutine object that was used for the task creation.` or something like this?
Sorry, I'm not a native speaker. Don't insist on the exact wording but you've got my idea.

Reply:
I'd just say this: "Return the coroutine object wrapped by the Task".